### ISSUE TITLE
GNOME bar

### ISSUE THREAD

**Issue by anasik** [btd1337/Stupendously-Hot-Charmander#13]
The top bar needs to be a lighter shade of gray, the applet icons are too solid and too white. 
If you look closely, you'd see that in the original concept:

- The applet icons didn't have a fill, and had really thin lines that were a soft shade of gray. Also, they had more padding. (This can be achieved by simply making them smaller.)
- There was no battery indicator, but we have seen enough to know what the battery indicator needs to be like. I personally believe that a vertical one doesn't work at all. I'd rather have a horizontal one. It goes without saying that it'd have the same line width and color as the rest of the icons
- It showed a messaging dropdown. We don't have that messenger but we do have a "mail" icon by default and that's the one that needs to change. currently it shows a "bell" that would be better suited for a "notifications" dropdown.

**Comment by rocketinventor** [comment]
@anasik I was also thinking that the icons weren't quite right and were different than the concept. I seems like it would become quite time consuming to do that manually, though. Right?

**Comment by anasik** [comment]
Yes, but it would be worth it. Maybe we could try contacting the designer of the concept and see if he can send us some files for them which could then be converted and added to the theme?
